modules/outbound-domain-event-handler/src/application/handlers/party-info-callback-received.ts

```diff
@@ -14,16 +14,19 @@ export async function handlePartyInfoCallbackReceived(
         = PartyInfoCallbackReceivedDmEvt.CreateFromDomainEvent(message);
     try {
         const processPartyInfoCallbackMessageData: IProcessPartyInfoCallbackCmdEvtData = {
-            key: partyInfoCallbackReceived.getKey(),
-            partyResult: partyInfoCallbackReceived.getPartyResult(),
+            bulkId: partyInfoCallbackReceived.getKey(),
```

Comment:
if you are changing `key` -> `bulkId`.....then shouldn't this be `bulkTransactionId`?